app/editor/src/features/content/tool-bar/sections/filter/DateRangeSection.tsx

```diff
@@ -27,12 +27,13 @@ export const DateRangeSection: React.FC<IDateRangeSectionProps> = () => {
         placeholderText="mm/dd/yyyy"
         selected={!!filterAdvanced.startDate ? new Date(filterAdvanced.startDate) : undefined}
         width="8em"
-        onChange={(date) =>
-          storeFilterAdvanced({
-            ...filterAdvanced,
+        onChange={(date) => {
+          storeFilter((filter) => ({ ...filter, timeFrame: 0 }));
```

Comment:
The auto fetch only fires when the standard filter is updated.  Not sure how this ever worked.